blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_3_materials/formula_3_23.py

```diff
@@ -0,0 +1,48 @@
+"""Formula 3.23 from NEN-EN 1992-1-1+C2:2011: Chapter 3 - Materials."""
+# pylint: disable=arguments-differ
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.formula import Formula
+from blueprints.type_alias import MM, MPA
+
+
+class Form3Dot23TensileBendingStrength(Formula):
```

Comment:
FlexuralTensileStrength

Here's the 2004 English version from our archives:
![image](https://github.com/Blueprints-org/blueprints/assets/145465456/04b4f601-dc10-4572-a3fd-bf949366e6a2)